docs/gatsby-config.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
       options: {
         alias: {
           '@primer/components': path.resolve(__dirname, '../src'),
```

Comment:
Can we remove @primer/components?

Reply:
We'll need this for a small transition period while Doctocat still references `@primer/components`